ark/phenotyping/pixel_cluster_utils.py

```diff
@@ -230,8 +231,9 @@ def smooth_channels(fovs, tiff_dir, img_sub_folder, channels, smooth_vals):
                    chan_out, check_contrast=False)
 
 
-def filter_with_nuclear_mask(fovs, tiff_dir, seg_dir, channel,
-                             img_sub_folder=None, exclude=True):
+def filter_with_nuclear_mask(fovs: List, tiff_dir: str, seg_dir: str, channel: str,
+                             nuc_seg_suffix: str = "nuclear", img_sub_folder: str = None,
```

Comment:
I think it would be better to have `nuc_seg_suffix: str="_nuclear.tiff"` here (to handle image files with different extensions).